Makefile

```diff
@@ -321,34 +352,38 @@ mec-tools/mec_encrypt: mec-tools/Makefile
 	git submodule update
 	make -C mec-tools
 
+mec-tools/mec_csum_outer: mec-tools/Makefile
+	git submodule update
+	make -C mec-tools
+
 # using function calls to build rules with actions is kind of a hack,
 # which is why these are all on oneline.
 
 # Insert the new firmware into the FL2 file
-# $1 = encoded EC firmware
+# $1 = repacked (csums fixed and optionally re-encrypted) EC firmware
 # $2 = FL2 filename
 define rule_fl2_patch
-    $(2): $(1) ; ./scripts/slice.insert $(1).slice $(1) $(2)
+    $(2): $(1) ./scripts/slice.insert ; ./scripts/slice.insert $(1).slice $(1) $(2)
```

Comment:
"should not change the contents of the resultant output." True, but we can elimiate this last bit of uncertanty also by simply rerunning the script if it changed.

I added this for "./scripts/slice.insert" as well when I was working on `copyFL2` which output did change. Does that work for you?